app-authz-photoz/photoz-restful-api/src/main/resources/photoz-restful-api-authz-service.json

```diff
@@ -115,7 +115,6 @@
       "logic": "POSITIVE",
       "decisionStrategy": "UNANIMOUS",
       "config": {
-        "defaultResourceType": "[\"http://photoz.com/album\"]",
```

Comment:
Without this line removed, the permission doesn't seem to take any effect. Not sure why but since this field doesn't seem to be exposed via UI, I think it's safe to remove. Or is it a bug in KC? @pedroigor?

Reply:
Probably not a bug because we have several tests in our test suite with higher coverage than what we do here. Including the same application.

In theory, this line should impact how permissions are evaluated due to the fact that you are defining permission for resources with a specific type. Without it, the server won't be able to enforce access because permissions are not going to match newly created resources. In this case, albums.

Let me take a look.

Reply:
@vmuzikar It is OK. The permission changed to a be a `scope` permission, so that attribute does not make sense.